### PR TITLE
Updates to generated webpack.config.js

### PR DESCRIPTION
**Pull request by petermbenjamin**
- Replace relative paths with 'path.join()'
- Add devServer for webpack-dev-server

### CODE REVIEW

File: webpack/src/utils.ts [hunk 1]
```diff
@@ -105,7 +106,10 @@ export function getWebpackConfig () {
       resolve: {
         extensions: ['.json', '.js', '.jsx', '.css']
       },
-      devtool: 'source-map'
+      devtool: 'source-map',
+      devServer: {
+        publicPath: path.join('/${bundlePath}/')
```
Comment (jeremyrajan) [inline webpack/src/utils.ts]: Why not just, `publicPath: ${bundlePath}` ? :)

**Comment by jeremyrajan** [comment]
Hey @petermbenjamin,

Thank you for the PR :). Tiny comment, otherwise all good. I will give it a shot locally and push it thru.

Cheers,
Jeremy

File: webpack/src/utils.ts (hunk 1 above)
Comment (petermbenjamin) [inline webpack/src/utils.ts]: @jeremyrajan - great question.
The [webpack.js.org](https://webpack.js.org) documentation explicitly states that `devServer.publicPath` **must** start and end with a `/`.

Ref: https://webpack.js.org/configuration/dev-server/#devserver-publicpath-

Comment (petermbenjamin) [inline webpack/src/utils.ts]: I had tried it in the past with just `publicPath: 'dist'` and not `publicPath: '/dist/'`, but webpack-dev-server would not serve the bundle in the browser.

Burnt about 1-2 hours of debugging time trying to figure this one out.

Comment (jeremyrajan) [inline webpack/src/utils.ts]: Awesome! sounds good. Thank you :)

**Pull request merged**